server/common/ZabbixAPI.cc

```diff
@@ -1355,7 +1355,7 @@ void ZabbixAPI::pushTriggersHostid(JSONParser &parser,
 	} else  {
 		for (int i = 0; i < numElem; i++) {
 			startElement(parser, i);
-			pushUint64(parser, itemGroup, "hostid", itemId);
+			pushString(parser, itemGroup, "hostid", itemId);
```

Comment:
The dummy data in another side of this block should also be replaced with the string type.

```diff
diff --git a/server/common/ZabbixAPI.cc b/server/common/ZabbixAPI.cc
index 221e384..152418b 100644
--- a/server/common/ZabbixAPI.cc
+++ b/server/common/ZabbixAPI.cc
@@ -1350,7 +1350,7 @@ void ZabbixAPI::pushTriggersHostid(JSONParser &parser,
        startObject(parser, "hosts");
        int numElem = parser.countElements();
        if (numElem == 0) {
-               const uint64_t dummyData = 0;
+               string dummyData;
                itemGroup->addNewItem(itemId, dummyData, ITEM_DATA_NULL);
        } else  {
                for (int i = 0; i < numElem; i++) {
```

Otherwise updating triggers will fail:

```
[DBG] <HatoholException.cc:49> HatoholException: <ItemTable.cc:72> ASSERTION failed: [*groupType0 == *groupType1] : ItemGroupTypes unmatched
[ERR] <ArmZabbixAPI.cc:262> Error on update: <ItemTable.cc:72> ASSERTION failed: [*groupType0 == *groupType1] : ItemGroupTypes unmatched
 2
```